detection-rules/bec_invoice_fraud_30d_domains.yml

```diff
@@ -0,0 +1,70 @@
+name: "BEC/Fraud - Invoice from sender, reply-to or return-path domain less than 30d old."
+description: "This rule checks for invoicing content from a sender, reply-to domain or return-path domain less than 30d old. It also checks the body or the OCR'd screenshot for key words commonly abused in fraudulent invoicing attacks. "
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  
+  // reply to domain that's less than 30d old and doesn't match the sender
+  and (
+    (
+      length(headers.reply_to) > 0
+      and all(headers.reply_to,
+              beta.whois(.email.domain).days_old <= 30
+              and .email.email != sender.email.email
+      )
+    )
+  
+    // or the return path or sender domain is less than 30d old 
+    or beta.whois(headers.return_path.domain).days_old <= 30
+    or beta.whois(sender.email.domain).days_old <=30
+  )
+  
+  // invoicing with high confidence
+  and any(ml.nlu_classifier(body.current_thread.text).tags,
+          .name == "invoice" and .confidence == "high"
+  )
+  
+  // commonly abused brands in body
+  and (
+    strings.ilike(body.current_thread.text,
+                  "*mcafee*",
+                  "*norton*",
+                  "*geek squad*",
+                  "*paypal*",
+                  "*ebay*",
+                  "*symantec*",
+                  "*best buy*",
+                  "*lifelock*",
+                  "*virus*"
+    )
+  
+    // commonly abused brand logo
+    or any(ml.logo_detect(beta.message_screenshot()).brands,
+           .name in ("PayPal", "Norton", "GeekSquad", "Ebay")
+    )
+  
+    // check message screenshot ocr for commonly abused brands
+    or any(file.explode(beta.message_screenshot()),
+           1 of (
+             strings.icontains(.scan.ocr.raw, "geek squad"),
+             strings.icontains(.scan.ocr.raw, "lifelock"),
+             strings.icontains(.scan.ocr.raw, "best buy"),
+             strings.icontains(.scan.ocr.raw, "mcafee"),
+             strings.icontains(.scan.ocr.raw, "norton"),
+             strings.icontains(.scan.ocr.raw, "ebay"),
+             strings.icontains(.scan.ocr.raw, "paypal"),
+             strings.icontains(.scan.ocr.raw, "virus"),
+           )
+    )
+  )
+attack_types:
+  - "BEC/Fraud"
```

Comment:
just confirming, is this not callback phishing given the brands were looking for?